test/mpi/pt2pt/testlist.def

```diff
@@ -47,3 +47,9 @@ huge_dupcomm 2
 dtype_send 2
 recv_any 2
 irecv_any 2
+precv_anysrc 2 arg=-msg_size=64 timeLimit=60
```

Comment:
Seems like a little overkill to split message size out like this. Can you just iterate over these inside the test case? You can add debug output to show when each iteration is complete so it's easy to investigate when there is a failure.